homeassistant/components/mqtt/cover.py

```diff
@@ -281,14 +281,23 @@ def tilt_message_received(msg):
                 )
 
             if payload.isnumeric() and (
-                self._config[CONF_TILT_MIN]
-                <= int(payload)
-                <= self._config[CONF_TILT_MAX]
+                (
+                    self._config[CONF_TILT_MIN]
+                    <= int(payload)
+                    <= self._config[CONF_TILT_MAX]
+                )
+                or (
+                    self._config[CONF_TILT_MAX]
+                    <= int(payload)
+                    <= self._config[CONF_TILT_MIN]
+                )
             ):
 
                 level = self.find_percentage_in_range(float(payload))
                 self._tilt_value = level
                 self.async_write_ha_state()
+            else:
+                _LOGGER.warning("Payload '%s' is not numeric", payload)
```

Comment:
Since is also used to indicate the payload is not in range we either need to change it to something like this:
```suggestion
                _LOGGER.warning("Payload '%s' is not numeric or out of range", payload)
```